src/mappings/nabla/backstopPoolEventHandler.ts

```diff
@@ -143,6 +151,12 @@ export async function handleMint(
         event.amountPrincipleDeposited,
         backstopPool
     )
+    addBackstopLP(
```

Comment:
On this implementation, this is how we keep track of the LP tokens per pool and user and avoid storing in the database.